### PR TITLE
Homepage

### PR DESCRIPTION
**Pull request by timuric**

### CODE REVIEW

File: docs/index.mdx [hunk 1]
```diff
@@ -1,26 +1,57 @@
 ---
-title: Saleor Documentation
-slug: /
+title: Saleor docs
```
Comment (peelar) [inline docs/index.mdx:2]: **suggestion (blocking)**: Please put titles in https://en.wikipedia.org/wiki/The_Chicago_Manual_of_Style.

**style (non-blocking)**: I think "Saleor Documentation" is more professional.

File: docs/index.mdx [hunk 2]
```diff
@@ -1,26 +1,57 @@
...
-<!-- ## Develop -->
+<Tiles />
 
-## Explore
+Learn about Saleor using tutorials, API reference, and developer guides.
 
 <CardGrid>
 
-[**![](./assets/icons/getting-started.svg) Overview** Get started with Saleor.](overview/introduction.mdx)
+    [<Icon32 name={"play"} /> **Quickstart** Explore tutorials to get up and running quickly](quickstart/cloud.mdx)
```
Comment (peelar) [inline docs/index.mdx:28]:
```suggestion
    [<Icon32 name={"play"} /> **Quickstart** Explore tutorials to get up and running quickly.](quickstart/cloud.mdx)
```

File: docs/index.mdx [hunk 3]
```diff
@@ -1,26 +1,57 @@
...
-[**![](./assets/icons/before-start.svg) Saleor CLI** Interact with Saleor the way you like it.](cli/overview.mdx)
+    [<Icon32 name={"buildApp"} /> **Build Apps** Build integrations and dashboard apps.](developer/app-store/overview.mdx)
 
 </CardGrid>
+
+<Block title="Core concepts" >
+    <Chapters >
+        [<Tag size={16} strokeWidth={1.5}/> **Products** _Configure catalog._](developer/products/overview.mdx)
+
+        [<PackageOpen size={16} strokeWidth={1.5}/> **Checkout and Orders** _Business logic of checkout_](developer/checkout/overview.mdx)
```
Comment (peelar) [inline docs/index.mdx:42]:
```suggestion
        [<PackageOpen size={16} strokeWidth={1.5}/> **Checkout and Orders** _Business logic of checkout._](developer/checkout/overview.mdx)
```

File: components/Apps/Apps.jsx [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+        <Shell color={"rgb(223, 110, 44)"} title={"Avatax"}>
+          <Taxes />
+        </Shell>
+        <Shell color={"rgb(39, 180, 118)"} title={"CMS"}>
+          <CMS />
+        </Shell>
+        <Shell color={"rgb(137, 82, 255)"} title={"CRM"}>
+          <CRM />
+        </Shell>
+        <Shell color={"rgb(59, 213, 121)"} title={"Data importer"}>
```
Comment (peelar) [inline components/Apps/Apps.jsx:46]:
```suggestion
        <Shell color={"rgb(59, 213, 121)"} title={"Data Importer"}>
```

File: components/Apps/Apps.jsx [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+  );
+};
+const Apps = () => {
+  return (
+    <div className={s.root}>
+      <div className={s.container}>
+        <Shell color={"rgb(10, 191, 83)"} title={"Adyen"}>
+          <Adyen />
+        </Shell>
+        <Shell color={"rgb(223, 110, 44)"} title={"Avatax"}>
```
Comment (peelar) [inline components/Apps/Apps.jsx:37]:
```suggestion
        <Shell color={"rgb(223, 110, 44)"} title={"AvaTax"}>
```

File: components/Apps/Apps.jsx [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+        <Shell color={"rgb(37, 195, 101)"} title={"Twilio Segment"}>
+          <Segment />
+        </Shell>
+        <Shell color={"rgb(95, 58, 199)"} title={"Slack"}>
+          <Placeholder />
+        </Shell>
+        <Shell color={"rgb(199, 159, 58)"} title={"SMTP"}>
+          <Notification />
+        </Shell>
+        <Shell color={"rgb(99, 91, 255)"} title={"Stipe"}>
```
Comment (peelar) [inline components/Apps/Apps.jsx:70]:
```suggestion
        <Shell color={"rgb(99, 91, 255)"} title={"Stripe"}>
```

File: components/Apps/Apps.jsx [hunk 7]
```diff
@@ -0,0 +1,81 @@
...
+          <Stripe />
+        </Shell>
+        <Shell color={"rgb(86, 189, 104)"} title={"Taxjar"}>
```
Comment (peelar) [inline components/Apps/Apps.jsx:73]:
```suggestion
        <Shell color={"rgb(86, 189, 104)"} title={"TaxJar"}>
```

**Review by peelar — CHANGES_REQUESTED** [review]
just some tiny spelling errors, etc. 

beyond that, looks awesome 👏

File: docs/index.mdx (hunk 1 above)
Comment (timuric) [reply docs/index.mdx:2]: I started correcting that in navigation, but there are leftovers

**Review by krzysztofwolski — APPROVED** [review]

File: components/Apps/Apps.jsx [hunk 8]
```diff
@@ -0,0 +1,129 @@
...
+        <Shell
+          color={"rgb(10, 191, 83)"}
+          title={"Adyen"}
+          href={"developer/app-store/apps/adyen/overview"}
+        >
+          <Adyen />
+        </Shell>
+        <Shell
+          color={"rgb(223, 110, 44)"}
+          title={"Avatax"}
```
Comment (peelar) [inline components/Apps/Apps.jsx:44]:
```suggestion
          title={"AvaTax"}
```

**Review by peelar — APPROVED** [review]

**Pull request merged**